setup.py

```diff
@@ -1,77 +1,39 @@
-#!/usr/bin/env python
+#! /usr/bin/env python
 # -*- coding: utf-8 -*-
 '''
 The setup script for SaltPyLint
 '''
 # pylint: disable=file-perms,wrong-import-position
 from __future__ import absolute_import, with_statement
-import io
 import os
-import sys
 
-SETUP_KWARGS = {}
-USE_SETUPTOOLS = False
 
-# Change to salt source's directory prior to running any command
-try:
-    SETUP_DIRNAME = os.path.dirname(__file__)
-except NameError:
-    # We're most likely being frozen and __file__ triggered this NameError
-    # Let's work around that
-    SETUP_DIRNAME = os.path.dirname(sys.argv[0])
-
-
-if SETUP_DIRNAME != '':
-    os.chdir(SETUP_DIRNAME)
-
-SALT_PYLINT_REQS = os.path.join(os.path.abspath(SETUP_DIRNAME), 'requirements.txt')
+BASE_DIR = os.path.dirname(__file__)
 
+INSTALL_REQUIRES = [
+    'PyLint>=1.7.1',
+    'pycodestyle>=2.3.1',
+    'modernize>=0.5'
+]
 
-def _parse_requirements_file(requirements_file):
-    parsed_requirements = []
-    with open(requirements_file) as rfh:
-        for line in rfh.readlines():
-            line = line.strip()
-            if not line or line.startswith(('#', '-r')):
-                continue
-            parsed_requirements.append(line)
-    return parsed_requirements
+setup_kwargs = {}  # pylint: disable=invalid-name
```

Comment:
Per PEP8 _"constants"_ should be all caps however this is not a "constant" as it is modified at least once after declaration (hence the suppression). PyLint assumes that all module level variables are "constants". This is lower case so as not to imply the data will not change.

However, if the coding conventions used here disagree I'm happy to uppercase this.